manifests/kustomize/env/aws/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
Can we also remove this owner file? so it won't trigger the bot when searching owners.

Reply:
Good catch. I removed it.